src/Microsoft.OData.Core/Value/ODataResourceValue.cs

```diff
@@ -0,0 +1,38 @@
+//---------------------------------------------------------------------
+// <copyright file="ODataResourceValue.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.OData
+{
+    /// <summary>
+    /// Represents the value of a resource (complex or entity).

```

Comment:
>Represents the value of a resource (complex or entity). [](start = 8, length = 55)

should probably clarify where this is used; i.e., in an annotation.

Reply:
#Resolved.